cpp/thirdparty/versions.txt

```diff
@@ -101,9 +101,8 @@ ARROW_RAPIDJSON_BUILD_VERSION=232389d4f1012dddec4ef84861face2d2ba85709
 ARROW_RAPIDJSON_BUILD_SHA256_CHECKSUM=b9290a9a6d444c8e049bd589ab804e0ccf2b05dc5984a19ed5ae75d090064806
 ARROW_RE2_BUILD_VERSION=2022-06-01
 ARROW_RE2_BUILD_SHA256_CHECKSUM=f89c61410a072e5cbcf8c27e3a778da7d6fd2f2b5b1445cd4f4508bee946ab0f
-# 1.1.9 is patched to implement https://github.com/google/snappy/pull/148 if this is bumped, remove the patch
```

Comment:
Is this comment still true? The patch above doesn't look like it's related, but I might be missing something!

Reply:
This was about our infamous 2 char sanppy fix that took forever to merge^^ but it is merged now and I have tagged the related issues in the description body :) (the patch was likely already removed as I could not find it or it was referencing the version we host in the artifactory?)